src/Microsoft.DotNet.XHarness.iOS.Shared/Hardware/SimulatorLoader.cs

```diff
@@ -462,6 +462,32 @@ public ISimulatorDevice FindCompanionDevice(ILog log, ISimulatorDevice device)
             return _availableDevices.Single(v => v.UDID == pair.Companion);
         }
 
+        public async Task<(ISimulatorDevice Simulator, ISimulatorDevice? CompanionSimulator)> FindSimulators(TestTargetOs target, ILog log, int retryCount, bool createIfNeeded = true, bool minVersion = false)
```

Comment:
It seems that you don't use `retryCount` parameter. Do you need it?

Reply:
Also a good catch :)